pkg/executor/task_handle.go

```diff
@@ -77,3 +77,15 @@ func StopAndEraseOutput(handle TaskHandle) (errorCollection errcollection.ErrorC
 
 	return
 }
+
+// GetWaitChannel returns channel that will return result (any encountered error) of
+// Wait() method in provided handle.
+func GetWaitChannel(handle TaskControl) <-chan error {
```

Comment:
@skonefal as of now there is no reason to export this function therefore I disagree with exporting it. It can be easily exported when use-case appears.